cmd/ttn-lw-stack/commands/config.go

```diff
@@ -43,7 +44,8 @@ type Config struct {
 	JS               joinserver.Config        `name:"js"`
 	Console          console.Config           `name:"console"`
 	GCS              struct {
-		BasicStation cups.ServerConfig `name:"basic-station"`
+		gatewayconfigurationserver.Config `name:",squash"`
+		BasicStation                      cups.ServerConfig `name:"basic-station"`
```

Comment:
Can we make Basic Station part of the `gatewayconfigurationserver` config? Or what is the reasoning for this cohesion?

Reply:
The initial thinking was that each component is started separately by the stack binary instead of the GCS itself. I can see why this is inconsistent, and have corrected it accordingly.